scripts/upgrade.sh

```diff
@@ -171,6 +171,7 @@ verify_controlplane_versions() {
 }
 
 # Function to compare semantic versions
+# Needs to improve to be able to compare release candidates
```

Comment:
I don't think we need to handle the pre-release and build metadata portions; checking only major/minor/patch is fine.